opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -602,6 +602,18 @@ const computePlatformMappings = () => {
           date_seen: { type: 'date' },
         },
       },
+      user_confidence_level: {
+        properties: {
+          max_confidence: { type: 'integer' },
+          overrides: {
+            type: 'nested',
+            properties: {
+              entity_type: { type: 'text' },
+              max_confidence: { type: 'integer' },
+            },
```

Comment:
using `nested` top be able to query the overrides inside the array properly (like "give me users with overrides on Reports greater than 40").

Not needed now, might be in the future for administrative purposes.